app/src/main/java/com/example/trainingintive/viewmodels/ActivitiesViewModel.kt

```diff
@@ -3,20 +3,26 @@ package com.example.trainingintive.viewmodels
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import com.example.trainingintive.di.IoScheduler
+import com.example.trainingintive.di.UiScheduler
 import com.example.trainingintive.model.ActivityModel
 import com.example.trainingintive.repository.ActivityRepository
-import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
-import io.reactivex.rxjava3.schedulers.Schedulers
+import io.reactivex.rxjava3.core.Scheduler
 import javax.inject.Inject
 
-class ActivitiesViewModel @Inject constructor(private val activityRepository: ActivityRepository) : ViewModel() {
+class ActivitiesViewModel @Inject constructor(
+    private val activityRepository: ActivityRepository,
+    @IoScheduler private val ioScheduler: Scheduler,
+    @UiScheduler private val uiScheduler: Scheduler
```

Comment:
It will be easier to use if you create a one class containing all used Schedulers.